roles/core/pxe_stack/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 pxe_stack_enable_cloning: false
 pxe_stack_firewall_zone:
```

Comment:
not zone by default ?

Reply:
No, I added variable to make it more visible (I am using them a lot on Algoric, and found they were not enough visible and documented in the main documentation, I need to improve that).

Reply:
ok